code/modules/projectiles/guns/rifles.dm

```diff
@@ -2268,7 +2268,7 @@
 	burst_scatter_mult = SCATTER_AMOUNT_TIER_10
 	scatter_unwielded = SCATTER_AMOUNT_TIER_2
 	damage_mult = BASE_BULLET_DAMAGE_MULT + BULLET_DAMAGE_MULT_TIER_1
-	recoil = RECOIL_AMOUNT_TIER_4 + RECOIL_AMOUNT_TIER_5/5
+	recoil = RECOIL_AMOUNT_TIER_3 + RECOIL_AMOUNT_TIER_3/5
```

Comment:
while you're changing this instead of adding tier values like this (old val is 2 + 1/5(2.2), new is 3 + 3/5(3.6) ) either use a pre-existing RECOIL_AMOUNT_TIER or add one to the list e.g. RECOIL_AMOUNT_TIER_L23 or RECOIL_AMOUNT_TIER_2_5 in code/_DEFINES/weapon_stats.dm